src/index.js

```diff
@@ -91,17 +91,7 @@ export default class Snuffles {
         return res
       })
       .then(res =>
-        res.json().then(json => {
-          const headers = {}
-          for (let pair of res.headers.entries()) {
-            headers[pair[0]] = pair[1]
-          }
-          return {
-            status: res.status,
-            headers,
-            body: json
-          }
-        })
+        res.json().then(json => ({ status: res.status, body: json }))
```

Comment:
What do you think about passing the headers without transforming them?

```js
res.json().then(json => ({
  status: res.status,
  body: json,
  headers: res.headers
}))
```

This would still allow access to the headers without having incompatibilties

Reply:
...starts to look like we could use a [`Proxy`](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Proxy) 🙂